addons/mail/views/mail_template_views.xml

```diff
@@ -95,6 +98,20 @@
             </field>
         </record>
 
+        <record model="ir.ui.view" id="mail_template_form_confirm_delete">
```

Comment:
We generally put the ID first, as we globally know the model and it has less importance when reading from left to right. Also can be updated according to guidelines, like
```suggestion
        <record id="mail_template_view_form_confirm_delete" model="ir.ui.view" >
```